src/components/LogViewer/LogViewer.tsx

```diff
@@ -10,9 +10,10 @@ import styles from './LogViewer.module.css';
 
 interface ILogViewerProps {
   isStatic?: boolean;
-  data: string | string[];
   heightOffset?: number;
+  autofocusSearchBar?: boolean;
   customActions?: ReactNode[];
+  data: string | string[];
```

Comment:
please do not change the line position of `data` attribute in the interface definition